k9mail/src/main/java/com/fsck/k9/exceptions/ParseException.java

```diff
@@ -0,0 +1,9 @@
+package com.fsck.k9.exceptions;
+
+public class ParseException extends RuntimeException {
```

Comment:
There's no real benefit of introducing this dedicated exception class. Please use one of the common exceptions, i.e. `IllegalArgumentException`.